main.go

```diff
@@ -111,12 +112,84 @@ func setupBluez(name string) error {
 }
 
 func main() {
+
+	name, unitID = commands.GetDeviceName()
+	ethAddr, ethErr := commands.GetEthereumAddress(unitID)
+
+	// - To scan can bus and save local copy on autopi, with custom name:
+	//./edge-network -candump <baudrate> <cycle_count> <file_out>
+
+	// - To scan can bus and send dump to mqtt:
+	// ./edge-network -sendcandump <baudrate> <cycle_count> <chunk_size>
+
+	// - To scan can bus and send dump to mqtt AND save local copy on autopi:
+	// ./edge-network -sendcandump <baudrate> <cycle_count> <chunk_size> savelocal
+
 	if len(os.Args) > 1 {
 		// this is necessary for the salt stack to correctly update and download the edge-network binaries. See README
 		s := os.Args[1]
 		if s == "-v" {
 			log.Printf("Version: %s", Version)
 			os.Exit(0)
+		} else if len(os.Args) > 4 {
```

Comment:
let's change this to be implemented as a subcommand. There is an example in `scan_vin.go` in the root of the project.